README.md

```diff
@@ -1,2 +1,69 @@
 # YNAP-Syllabus
 YNAP's Curriculum for Coding Mastery: Creating Valuable and Functional Software
+
+YNAP incorporates testing from day one as the strategy to ensure that learning is continually validated and reinforced. 
+
+With YNAP, testing is not an afterthought as is too often the case. Testing is a core part of the learning process, instilling best practices from the outset.
+
+### Blocks 1-2: Basics of JavaScript and Introduction to Testing
+- **Day 1-2:** Introduction to JavaScript: Syntax, Variables, Data Types
+- **Day 3:** Basic Testing Concepts: Writing Your First Tests
+- **Day 4-5:** Control Structures in JavaScript and Testing Them
+- **Project Start:** Simple Calculator (console-based) with Tests
+
+### Blocks 3-4: Functions, Web Concepts, and Testing Integration
+- **Day 1-2:** Functions in JavaScript and Testing Functions
+- **Day 3-4:** Introduction to HTML/CSS and JavaScript Integration
+- **Day 5:** Writing Tests for Web Interfaces
+- **Project Continuation:** Enhance Calculator with Web Interface and Tests
+
+### Blocks 5-6: Advanced JavaScript Features with Test-Driven Development
+- **Day 1-2:** Arrays, Objects in JavaScript, and Testing Complex Data Structures
+- **Day 3-4:** Asynchronous Programming in JavaScript and Testing Asynchronous Code
+- **Day 5:** Error Handling, Debugging Techniques, and Testing Error Scenarios
+- **Project Addition:** Todo List Feature with Tests
+
+### Blocks 7-8: Introduction to Supabase, Backend Testing
+- **Day 1-2:** Setting up Supabase and Writing Basic Backend Tests
+- **Day 3-4:** Database Operations and Writing Tests for CRUD Operations
+- **Day 5:** Integrating Supabase with JavaScript with Testing
+- **Project Expansion:** User Authentication in Todo List App with Tests
```

Comment:
It might be good to instead introduce a generalized backend instead of a PaaS. I think it would be extremely overwhelming for a new dev to be thrown in the deep end with no understanding of backends already as well as the general lack of help with Supabase.